nextflow.config

```diff
@@ -12,6 +12,23 @@ params {
     // Input options
     input                      = null
 
+    // Taxonomy classification options
+    taxonomy_mmseqs_classification_off      = false
```

Comment:
Please use the same naming schema as the rest of the pipeline: `--run_taxonomic_classification`. If you think (which I think was implied by your code/comments further up) other classifications tools may be added, might make sense to have a `--taxonomic_classification_tool 'mmseqs2'` flag

Reply:
I would suggest only 'skip_taxonomic_classification == false' as the taxonomic classification is run by default

Reply:
ok so that is now changed, the taxnomic classification is now turned of by default and can be run with `run_taxonomic_classification`